stage0/src/asm/boot.s

```diff
@@ -73,10 +73,59 @@ _protected_mode_start:
     # Set up a basic stack, as we may get interrupts.
     mov $stack_start, %esp
 
+    # Zero-out all the page tables: start address in EDI, value in AL, count in ECX.
+    mov $4906, %ecx           # each table is exactly 4096 bytes long.
```

Comment:
Typo here. Or, I'd rather use `$0x1000` as that's cleaner.

Reply:
Thanks, good catch. Fixed.